graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -90,20 +99,17 @@ def _get_filter_selectivity(
             schema_graph, statistics, location_name, filter_info.fields
         )
 
-        # Assumption: The selectivity is proportional to the number of entries in the collection
-        # This will not hold in case of duplicates.
+        # Assumption: the selectivity is proportional to the number of entries in the collection.
         if _is_absolute(selectivity_per_entry_in_collection):
             result_selectivity = Selectivity(
                 kind=ABSOLUTE_SELECTIVITY,
                 value=float(collection_size) * selectivity_per_entry_in_collection.value
             )
         elif _is_fractional(selectivity_per_entry_in_collection):
             result_selectivity = Selectivity(
-                kind=selectivity_per_entry_in_collection.kind,
+                kind=FRACTIONAL_SELECTIVITY,
                 value=min(float(collection_size) * selectivity_per_entry_in_collection.value,
                           1.0)
-                # The estimate may be above 1.0 in case of duplicates in the collection
```

Comment:
Why delete this comment but preserve the `min` logic?

Reply:
Reverted.